Source/Core/Core/IPC_HLE/WII_IPC_HLE_Device_usb.cpp

```diff
@@ -1308,6 +1308,9 @@ void CWII_IPC_HLE_Device_usb_oh1_57e_305::CommandDisconnect(u8* _Input)
 	DEBUG_LOG(WII_IPC_WIIMOTE, "  Reason: 0x%02x", pDiscon->reason);
 
 	Host_SetWiiMoteConnectionState(0);
+	// TODO: If someone wants to be fancy we could also figure out what the values for pDiscon->reason mean
+	// and display things like "Wiimote %i disconnected due to inactivity!" etc.
+	Core::DisplayMessage(StringFromFormat(_trans("Wiimote %i disconnected by emulated software!"), (pDiscon->con_handle & 0xFF) + 1), 3000);
```

Comment:
This should likely be it's own static class function that displays this. e.g. DisplayDeconnectMessage. That way if displaying reasons is tackled later, it's all confined in one area.